install/kubernetes/helm/istio/files/injection-template.yaml

```diff
@@ -46,6 +46,11 @@ initContainers:
     privileged: true
     {{- end }}
   restartPolicy: Always
+  env:
+  {{- if (isset .ObjectMeta.Annotations `sidecar.istio.io/inboundInterceptionSplit`) }}
+  - name: INBOUND_CAPTURE_PORT
+    value: {{ valueOrDefault .Values.global.inboundInterceptionPort `15006` }}
+  {{- end }}
```

Comment:
I'm not sure there is a user case for user top care that 15006 is used - when and why would a user change it ? 
Leave it as 15006, if we find a use case for customizing it we can add it later. 

Also the line above should check if either the global or annotation is set.

Reply:
yes, the env will be set if either the global or annotations are set.

Reply:
hard code 15006 here

Reply:
Well, that's not 'hard coded' since the inject template is editable.

Reply:
But if you have it here - which implies user can change it by edit on the template - you also need to pass it as a param to 
pilot, otherwise pilot generates rules for the wrong port.

Really we're spending too much time on details that almost nobody will use, and increasing complexity without good use cases...

@rshriram  - how about getting this PR without option to change the capture port, it'll be alpha anyways - and we can 
figure out how to handle it in 1.3, maybe with the proposed jsonpath overrides or some other less-intrusive mode ?